sdk/nodejs/appservice/zMixins.ts

```diff
@@ -470,3 +474,14 @@ export function getResourceGroupNameAndLocation(
     const getResult = resourceGroupName.apply(n => core.getResourceGroup({ name: n }));
     return { resourceGroupName, location: getResult.location };
 }
+
+/** @internal */
+export function mergeBindings(additionalBindings: BindingDefinition[], bindings?: pulumi.Input<BindingDefinition[]>) : pulumi.Input<BindingDefinition[]>{
+    let merged : pulumi.Input<BindingDefinition[]>;
+    if (bindings) {
+        merged = pulumi.output(bindings!).apply(b => [...b, ...additionalBindings]);
+    }else{
+        merged = additionalBindings;
+    }
+    return merged;

```

Comment:
consider rewriting this all as:

```ts
return bindings ? pulumi.output(bindings).apply(b => [...b, ...additionalBindings]) : additionalBindings;
```

Reply:
fixed in d783c138f08bff57325be4e7fb98e3a6933d5b8b